keras_nlp/src/models/vgg/__init__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2024 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from keras_nlp.src.models.vgg.vgg_backbone import VGGBackbone
```

Comment:
Ah I think the only reason we need import here is to register built in presets. Like [this](https://github.com/keras-team/keras-nlp/blob/master/keras_nlp/src/models/bert/__init__.py#L15-L23). We don't have any yet for VGG. I think we can remove this probably?